tcms/testruns/templates/testruns/get.html

```diff
@@ -0,0 +1,102 @@
+    <div class="card-pf card-pf-accented card-pf-aggregate-status">
+
+        <h2 class="card-pf-title" style="text-align: left">
+            <span class="fa fa pficon-topology"></span>{% trans 'Test plan' %}:
+            <a href="{% url 'test_plan_url_short' test_run.plan.pk %}">[{{ test_run.plan.pk }}] {{ test_run.plan.name }}</a>
+        </h2>
+
+        <h2 class="card-pf-title" style="text-align: left">
+            <span class="fa fa-shopping-cart"></span>{% trans 'Product' %}:
+            <a href="{% url 'testruns-search' %}?product={{ test_run.product_version.product_id }}" title="Search test runs of {{ test_run.product_version.product }}">{{ test_run.product_version.product }}</a>
+        </h2>
+
+        <h2 class="card-pf-title" style="text-align: left">
+            <span class="fa fa-search"></span>{% trans 'Product version' %}:
+            <a href="{% url 'testruns-search' %}?product={{ test_run.plan.product_id }}&product_version={{ test_run.plan.product_version_id }}" title="Search test runs of {{ test_run.plan.product_version.value }}">{{ test_run.plan.product_version.value }}</a>
+        </h2>
+
+        <h2 class="card-pf-title" style="text-align: left">
+            <span class="fa fa-cog"></span>{% trans 'Build' %}:
+            <a href="{% url 'testruns-search' %}?product={{ test_run.plan.product_id }}&product_version={{ test_run.plan.product_version_id }}&build={{ test_run.build_id }}" title="Search test runs of {{ test_run.build_id }}">{{ test_run.build }}</a>
+        </h2>
+
+        <h2 class="card-pf-title" style="text-align: left">
+            <span class="fa pficon-user"></span>{% trans 'Manager' %}:
+            <a href="{% url 'tcms-profile' test_run.manager.username %}">{{ test_run.manager.username }}</a>
+        </h2>
+
+        <h2 class="card-pf-title" style="text-align: left">
+            <span class="fa pficon-user"></span>{% trans 'Default tester' %}:
+            <a href="{% url 'tcms-profile' test_run.default_tester.username %}">{{ test_run.default_tester.username }}</a>
+        </h2>
+
+        {% comment %} TODO: buttons are still not working {% endcomment %}
+        {% comment %} TODO: better styling for buttons {% endcomment %}
+        <h2 class="card-pf-title" style="text-align: left">
+            {% if test_run.stop_date %}
+                <span class="fa fa-check-square"></span>{% trans 'Status' %}:
+                <span>{% trans "Finished" %}</span>
+                {% if perms.testruns.change_testrun %}
+                    <input type="button" class="set-running-button" value="Set to Running">
+                {% endif %}
+            {% else %}
+                <span class="fa fa-play"></span>{% trans 'Status' %}:
+                <span>{% trans "Running" %}</span>
+                {% if perms.testruns.change_testrun %}
+                    <input type="button" class="set-finished-button" value="Set to Finished">
+                {% endif %}
+            {% endif %}
```

Comment:
You can if/else the display only portion and then add the input element afterwards. 

For widget use bootstrap-switch button, see testplans/search.html